packages/modeling/src/operations/offsets/offsetGeom2.test.js

```diff
@@ -49,7 +74,7 @@ test('offset (options): offsetting of a simple geom2 produces expected offset ge
   t.true(comparePoints(pts, exp))
 
   // contract -
-  obs = offset({ delta: -0.5, corners: 'round' }, geometry)
+  obs = offset({ delta: -0.5, corners: 'round', segments: 0 }, geometry)
```

Comment:
segments = 0? is that allowed when 'round' corners are requested?

Reply:
Unfortunately, yes it is the default in V2, see [here](https://github.com/jscad/OpenJSCAD.org/blob/master/packages/modeling/src/operations/expansions/offsetGeom2.js#L19). I had to set `segments = 0` here to make this test pass.